### DISCUSSION TITLE
I have an issue where the notification setting on websites aren't permanent, even if the value of 'content.notifications.enabled' is 'true

### DISCUSSION THREAD

**Discussion by groggone** [qutebrowser/qutebrowser#7946] [Q&A]
This is a duplicate of [#7941](https://github.com/qutebrowser/qutebrowser/discussions/7941), the issue still hasn't been resolved. The workaround scripts in [#5233](https://github.com/qutebrowser/qutebrowser/issues/5233) are just mainly for some certain sites. I just want answers

**Comment by The-Compiler** [comment]
Well, as you say: this is a duplicate...

**Reply by groggone** [reply]
I know i just answers... 😭
